src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -36,53 +49,73 @@ const useGroupSetting = ({navigation, route}) => {
     }
   }, [route]);
 
-  const submitData = async (withNavigation = true, withLoading = true) => {
-    let changeImageUrl = '';
-    if (changeImage) {
-      if (withLoading) setIsLoading(true);
-      try {
-        const res = await uploadFile(`data:image/jpeg;base64,${base64Profile}`);
-        changeImageUrl = res.data.url;
-      } catch (e) {
-        setIsLoading(false);
-        return;
-      }
+  React.useEffect(() => {
+    if (selectedChannel) {
+      setUrlImage(selectedChannel?.channelPicture);
     }
+  }, [selectedChannel]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new useEffect hook has been added to update the channel image URL when the selected channel changes. This seems like a reasonable addition, but it's important to ensure that the `selectedChannel` variable is always up-to-date and that this hook doesn't introduce any unnecessary re-renders.

```diff
+ React.useEffect(() => {
+   if (selectedChannel) {
+     setUrlImage(selectedChannel?.channelPicture);
+    }
+ }, [selectedChannel]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->